cypress/support/step-definitions/pager-steps.js

```diff
@@ -23,6 +25,62 @@ Then('I am on 1st of {string} pages', (count) => {
   maxPages().should('have.text', ` of ${count}`);
 });
 
+Then('pagination {string} button is disabled', (button) => {
+  switch (button) {
+    case 'next':
+      paginationButton(TWO).should('have.attr', 'disabled');
+      break;
+    case 'last':
+      paginationButton(THREE).should('have.attr', 'disabled');
+      break;
+    case 'previous':
+      paginationButton(ONE).should('have.attr', 'disabled');
+      break;
+    case 'first':
+      paginationButton(ZERO).should('have.attr', 'disabled');
+      break;
+    default: throw new Error('There are only four pagination buttons');
+  }
+});
+
+Then('I click {string} pagination button', (button) => {
+  switch (button) {
+    case 'next':
+      paginationButton(TWO).click();
+      break;
+    case 'last':
+      paginationButton(THREE).click();
+      break;
+    case 'previous':
+      paginationButton(ONE).click();
+      break;
+    case 'first':
+      paginationButton(ZERO).click();
+      break;
+    default: throw new Error('There are only four pagination buttons');
+  }
+});
+
+Then('I click {string} pagination arrow', (arrow) => {
+  switch (arrow) {
+    case 'previousArrow':
+      nextArrow().parent().click();
+      previousArrow().parent().click();
+      break;
+    case 'nextArrow':
+      nextArrow().parent().click();
+      break;
+    default: throw new Error('There are only two pagination arrows');
+  }
+});
+
+Then('pagination buttons are disabled', () => {
+  const buttonsAmount = 4;
```

Comment:
could we take an amount by cypress `its()` command?

Reply:
`its()` return amount of elements, but for doesn't work with it